src/App.js

```diff
@@ -0,0 +1,91 @@
+import { BrowserRouter as Router, Link, Redirect, Route} from 'react-router-dom'
+import TransactionsByCategory from './Components/TrabsactionsByCategory'
+import Transactions from './Components/Transactions'
+import Operations from './Components/Operations'
+import { Component } from 'react'
+import React from 'react'
+import axios from 'axios'
+import './App.css'
+
+export class App extends Component {
+
+  constructor() {
+    super()
+    this.state = {
+      data:[], 
+      balance: "",
+      redirect:null
+    }
+  }
+
+  componentDidMount = async () =>{
+    await this.getData()
+    this.calculateBalance()
+  }
+
+  getData = async () =>{
+    try{
+      let transactions = await axios.get("http://localhost:3100/transactions")
+        await this.setState({data:transactions.data})
+        this.calculateBalance()
+    }
+    catch(err){
+      console.log(err)
+    }
+  }
+  calculateBalance = () =>{
+    let balance = 0 
+    this.state.data.forEach(t => balance += parseInt(t.amount))
```

Comment:
check [this](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/reduce) array method...